src/pages/create/OfferForm.tsx

```diff
@@ -259,8 +259,14 @@ const OfferForm = () => {
       configurations={[
         scopeStepConfiguration,
         typeAndThemeStepConfiguration,
-        calendarStepConfiguration,
-        locationStepConfiguration,
+        {
+          ...calendarStepConfiguration,
+          title: () => t(`create.calendar.title.${scope}`),
+        },
+        {
+          ...locationStepConfiguration,
+          title: () => t(`create.location.title.${scope}`),
+        },
```

Comment:
This can be reverted if you can use scope in the configurations below

Reply:
Yes correct, also did it for `additionalInformationStep`